backend/src/data/models/user/user.model.ts

```diff
@@ -4,6 +4,8 @@ import { Abstract } from '../abstract/abstract.model';
 class User extends Abstract {
   'email': string;
 
+  'full_name': string;
```

Comment:
Should be camelCase as `passwordHash` or `passwordSalt` instead of snake_case here